lib/views/widgets/button.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_ecommerce/utilities/constants/color_manager.dart';
+
+class Button extends StatelessWidget {
+  final String title;
+  final Function onPress;
+  const Button({
+    Key? key,
+    required this.title,
+    required this.onPress,
+  }) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      //style: ElevatedButtonThemeData(style: ),
+      onPressed: () {
+        onPress();
```

Comment:
You can add onPress to the function and implicitly link the parameters.
Like this:
onPressed: onPress.
Because it needs a voidCallback so don't need to make it an anonymous function. Just pass the function implicitly.